web/packages/teleterm/src/ui/ConnectMyComputer/connectMyComputerContext.tsx

```diff
@@ -275,6 +283,7 @@ export const ConnectMyComputerContextProvider: FC<{
     const shouldAutoStartAgent =
       isAgentConfigured &&
       canUse &&
+      !isNonCompatibleAgent &&
```

Comment:
What if we exported `isCompatibleAgent` instead? This way we would avoid the double negation here.

Reply:
Done